buildkite/scripts/tests/rosetta-integration-tests.sh

```diff
@@ -1,6 +1,26 @@
 #!/bin/bash
 set -eox pipefail
 
+# Function to collect logs (called on exit or at end of script)
+collect_logs() {
+    echo "========================= COLLECTING LOGS ==========================="
+    mkdir -p test_output/artifacts
+    
+    # application logs
+    cp daemon-stdout.log test_output/artifacts/ 2>/dev/null || echo "daemon-stdout.log not found"
```

Comment:
Why ignored stderr output?

Reply:
Also, we probably don't care about stdlogs given file logs is almost always a superset of it.